app/qml/form/components/calendar/MMDateTumbler.qml

```diff
@@ -26,72 +26,54 @@ Item {
   readonly property int calendarYearFrom: 1900
   readonly property int calendarYearTo: 2050
 
-  Rectangle {
+  function monthList() {
+    const monthList = Array(12).keys();
+    const getMonthName = (monthIndex) => Qt.locale().monthName(monthIndex)
+    return Array.from(monthList, getMonthName)
+  }
+
+  function yearList() {
+    var years = []
+    for (var i = root.calendarYearFrom; i <= root.calendarYearTo; ++i)
+      years.push(i)
+    return years
+  }
```

Comment:
Function definitions should be at the bottom of the file according to our code style